appcues/src/main/java/com/appcues/data/remote/NetworkRequest.kt

```diff
@@ -22,6 +23,8 @@ internal object NetworkRequest {
             exception.response()?.errorBody()?.source()?.let {
                 MoshiConfiguration.moshi.adapter(ErrorResponse::class.java).fromJson(it)
             }
+        } catch (exception: IOException) {
+            null
```

Comment:
discovered in tests, which sometimes mock responses with no body, that an `IOException` is raised for unexpected end of file - so just covering that case here as well - would be expected to only really happen in test cases, but in any case, we don't want to allow an exception to bubble up while just trying to parse an error message.